src/Analyzer.Core.BuiltInRuleTests/TestRunner.cs

```diff
@@ -74,7 +74,7 @@ private IEnumerable<int> GetAllFailedLines(IEvaluation evaluation)
         {
             if (!evaluation.Result?.Passed ?? false)
             {
-                yield return evaluation.Result.LineNumber;
+                yield return evaluation.Result.SourceLocation.LineNumber;
```

Comment:
Shouldn't this whole function be removed and its usage replaced by a statement using the new ReportsHelper.GetFailedResults function?

Reply:
yes it should